RealmSwift/Tests/ObjectSchemaTests.swift

```diff
@@ -24,7 +24,9 @@ class ObjectSchemaTests: TestCase {
 
     override func setUp() {
         super.setUp()
-        objectSchema = Realm().schema["SwiftObject"]
+        autoreleasepool {
+            self.objectSchema = Realm().schema["SwiftObject"]
```

Comment:
it's not "needed" per-se but it was a member before (this PR just moves setting it into the autoreleasepool) and it helps reduce code duplication